tcs-service/src/test/java/com/transformuk/hee/tis/tcs/message/RevalidationMessageListenerTest.java

```diff
@@ -65,4 +71,23 @@ public void shouldNotSendIfExhangeFalse() {
     verify(rabbitTemplate, never()).convertAndSend(any(String.class), any(String.class), any(ConnectionInfoDto.class));
   }
 
+  @Test
+  public void shouldSendMsgForCurrentPmUpdate() {
+    setField(revalidationMessageListener, EXCHANGE, EXCHANGE);
+    setField(revalidationMessageListener, ROUTING_KEY, ROUTING_KEY);
+
+    List<String> personIdStrs = Lists.newArrayList("1", "2", "3");
+    revalidationMessageListener.receiveMessageNightlyPmSync(personIdStrs);
+
+    verify(revalidationService, times(3))
+        .buildTcsConnectionInfo(personIdCaptor.capture());
+    verify(revalidationRabbitService, times(3))
+        .updateReval(connectionInfoDtoArguementCaptor.capture());
+
+    List<Long> personIds = personIdCaptor.getAllValues();
+    List<ConnectionInfoDto> connectionInfoDtos = connectionInfoDtoArguementCaptor.getAllValues();
+
+    assertThat(personIds).hasSize(3).contains(1l, 2l, 3l);
```

Comment:
`containsExactlyInAnyOrder()` verifies length & contents (optional)
something odd.  I expected sonar to complain about lower case `l` which can get confused with other characters.  `L` is the preferred case.

Reply:
Done, thanks.